packages/dashboard-core-plugins/src/linker/LinkerOverlayContent.scss

```diff
@@ -14,17 +14,38 @@
     $toast-outer-padding: $spacer-3;
 
     position: absolute;
-    right: $toast-outer-padding;
-    bottom: $toast-outer-padding;
     background: $modal-content-bg;
     max-width: 400px;
     color: $background;
     pointer-events: auto;
     box-shadow: $tooltip-box-shadow;
     z-index: $zindex-popover;
 
+    .btn-drag-handle {
+      cursor: grab;
+      border-radius: 0;
+      border: none;
+      position: absolute;
+      background: $gray-300;
+      color: $gray-600;
+      top: 50%;
+      left: 0;
+      height: 100%;
+      min-width: 1.5rem;
+      max-width: 2rem;
+      transform: translateY(-50%);
+
+      &:hover {
+        background-color: $gray-400;
+        color: $gray-700;
+      }
+    }
+
     .toast-body {
       padding: $toast-inner-padding;
+      position: relative;
+      margin-right: 10px;
+      left: 20px;
```

Comment:
Should use `margin-left` here instead of just `left`. Right now the div is actually bleeding out of the dialog on the right if you inspect element (can't tell because of padding/margin though).

Also should use `$spacer-2` and `$spacer-4` instead of 10px/20px.